### PR TITLE
Updates some module READMEs

### PR DESCRIPTION
**Pull request by KipHamiltons**
I am keen to remove darwin references from the codebase. These readmes had darwin references. I removed those, and spruced things up ever so slightly. The READMEs are by no means fully fixed. that's not the point of this PR.

### CODE REVIEW

File: module/input/SensorFilter/README.md [hunk 1]
```diff
@@ -16,8 +16,10 @@ The resulting filtered data will then be outputted as `message::input::Sensors`
...
--
+- `message::platform::ButtonLeftDown` when button pressed.
+- `message::platform::ButtonLeftUp` when button released.
+- `message::platform::ButtonMiddleDown` when button pressed.
+- `message::platform::ButtonMiddleUp` when button released.
+- `message::support::nusight::DataPoint` if log_level >= DEBUG, indicating if feet are down.
 
 ## Dependencies
-
-- The Darwin Hardware I/O module is required to provide the data to filter
```
Comment (ysims) [inline module/input/SensorFilter/README.md:23]: We still do need something that emits RawSensors? Gazebo, Webots or HardwareIO module.

File: module/input/SensorFilter/README.md [hunk 2]
```diff
@@ -16,8 +16,10 @@ The resulting filtered data will then be outputted as `message::input::Sensors`
 ## Emits
 
 - `message::input::Sensors` with filtered data from the input.
--
+- `message::platform::ButtonLeftDown` when button pressed.
+- `message::platform::ButtonLeftUp` when button released.
+- `message::platform::ButtonMiddleDown` when button pressed.
+- `message::platform::ButtonMiddleUp` when button released.
+- `message::support::nusight::DataPoint` if log_level >= DEBUG, indicating if feet are down.
```
Comment (ysims) [inline module/input/SensorFilter/README.md:23]: I think this is a `utility::nusight::graph`? not data point

Comment (JosephusPaye) [reply module/input/SensorFilter/README.md:23]: `graph` is a [helper function that creates and emits DataPoint](https://github.com/NUbots/NUbots/blob/c550271424d0c7742069e87dac371a49c1b0c5b6/shared/utility/nusight/NUhelpers.hpp#L60).

File: module/behaviour/skills/Getup/README.md [hunk 3]
```diff
@@ -11,15 +11,19 @@ Include this module to allow the robot to get up after it has fallen over.
 
 ## Consumes
 
-* `message::Sensors` containing the orientation matrix and accelerometer readings
-* `message::AllServoWaypointsComplete` indicates that a script has finished
+* `message::motion::ExecuteGetup` tells this module to call getup script
```
Comment (JosephusPaye) [inline module/behaviour/skills/Getup/README.md:14]: Seems fine, though why not use hyphen for lists, to match formatting in NUbook 😛 

```suggestion
- `message::motion::ExecuteGetup` tells this module to call getup script
```

**Review by JosephusPaye — APPROVED** [review]
LGTM

**Pull request merged**